packages/EasyCodingStandard/packages/FixerRunner/src/Application/FixerFileProcessor.php

```diff
@@ -167,23 +165,6 @@ public function processFile(SplFileInfo $fileInfo): string
         return $tokens->generateCode();
     }
 
-    public function processFileSecondRun(SplFileInfo $file): string
-    {
-        $this->prepareSecondRun();
-
-        return $this->processFile($file);
-    }
-
-    /**
-     * @return DualRunInterface[]|FixerInterface[]
-     */
-    public function getDualRunCheckers(): array
-    {
-        return array_filter($this->fixers, function (FixerInterface $fixer): bool {
-            return $fixer instanceof DualRunInterface;
-        });
-    }
-
```

Comment:
There are no second run fixers at the moment, so this code is really useless.
Also that shall be an experiment feature, only where it makes sense.